### ISSUE TITLE
Derivatives in BIDS format? 

### ISSUE THREAD

**Issue by mnarayan** [BIDS-Apps/CPAC#14]
Are there any plans in the works to output the derivatives such as regional timeseries or the nuisance regresssors used according to BIDS formatting?

**Comment by ccraddock** [comment]
Yes there are plans, but at this time we are not sure when we can get to this.

**Comment by ccraddock** [comment]
I have been slowly working on this, but I have a script that works OK now. It takes CPAC output directory and converts to something similar to BIDS derivatives format. I say similar because very little has been decided for the bids derivatives format. I tried to be consistent with what is already there and added new things that I think are consistent with the existing.

https://github.com/FCP-INDI/cpac_outputs_to_bids
